packages/common-scripts/examples/pw_lock/lock.ts

```diff
@@ -0,0 +1,391 @@
+import { LockScriptInfo, FromInfo, parseFromInfo, common } from "../../src";
+import {
+  Script,
+  CellProvider,
+  QueryOptions,
+  CellCollector as CellCollectorInterface,
+  Cell,
+  HexString,
+  PackedSince,
+  OutPoint,
+  values,
+  WitnessArgs,
+  core,
+  utils,
+  CellDep,
+} from "@ckb-lumos/base";
+import {
+  Options,
+  TransactionSkeletonType,
+  createTransactionFromSkeleton,
+} from "@ckb-lumos/helpers";
+import { getConfig, Config, initializeConfig } from "@ckb-lumos/config-manager";
+import { Reader, normalizers } from "ckb-js-toolkit";
+import { Set } from "immutable";
+import keccak, { Keccak } from "keccak";
+
+const { ScriptValue } = values;
+
+// https://github.com/lay2dev/pw-lock pw-lock is a lock script which uses secp256k1_keccak256 algorithm.
```

Comment:
Let's include the specific commit for the pw-lock used here, in case there are changes made to pw-lock.

Reply:
Updated